src/pymail

```diff
@@ -25,56 +26,74 @@ install_path = "/home/martijn/Coding/git/PyMail/src/"
 # Functions
 
 def send_email_from_terminal(use_sys_argv):
-    """Send an email"""
-    if use_sys_argv == False:
-        to_addr = raw_input("Receiver address = ")
-        subject = raw_input("Subject = ")
-        body = raw_input("Body = ")
-    elif use_sys_argv == True:
-        to_addr = sys.argv[2]
-        subject = sys.argv[3]
-        body = sys.argv[4]
+    """Set the mail-components from sysargv or from user input
+    depending on if use_sys_argv is True or False. Send mail after
+    setting variables.
+    """
+
+    if use_sys_argv:
+    	# sysargv 2, 3, and 4
+        to_addr, subject, body = [sys.argv[x] for x in range(2,5)]
```

Comment:
This is actually a good example of what list comprehensions are supposed to do. You may pull the 'ol argument that it makes it less readable, but according to the Zen of Python (PEP 20):

> Although practicality beats purity.

And this is damn practical